src/components/admin/MainIssueController.jsx

```diff
@@ -156,8 +157,8 @@ export default class MainIssueController extends FalcorController {
               return false;
             }
           }
-          const absoluteUrlRegex = /<a.*?href\s*?=\s*?["'](?!http)(.*?)["'] *?>/;
-          if (absoluteUrlRegex.test(article.html)) {
+          const absoluteUrlRegex = /<a.*?href\s*?=\s*?["']http(.*?)["'] *?>/;
```

Comment:
we don't want this duplicated twice for sure

Reply:
actually, I tried passing the absoluteurlregex as one of the arguments for the function so that I wouldn't have to duplicate it but it doesn't work for some reason.

Reply:
Well for one it should work, not sure what went wrong, but I'm super okay with hardcoding the regex inside the function, just as I talked about above return the url, and return `null` or something if all is good and no relative url was found